code/modules/research/nodes/robotics.dm

```diff
@@ -272,6 +272,7 @@
 
 	unlocks_designs = list(
 							/datum/design/research/item/mecha/weapon/laser_heavy,
+							/datum/design/research/item/mecha/weapon/auto_laser,
```

Comment:
No

Reply:
Why shouldn't this weapon be made available?

Reply:
Balance and lore, this should be hand crafted out of 4 lasers not printed

Reply:
Uh. fair enough.